tests/acceptance/route-announcer-test.js

```diff
@@ -1,71 +1,56 @@
+module('Acceptance | route announcer', function(hooks) {
+  setupApplicationTest(hooks);
 
-  beforeEach(function() {
-    application = startApp();
-    this.service = application.__container__.lookup('service:announcer');
-  });
-
-  afterEach(function() {
-    run(application, 'destroy');
-  });
+  module('setting different aria properties', function(hooks) {
 
-  describe("setting different aria properties", function() {
-    beforeEach(function() {
-      visit('/');
-      return click('.spec-announce-btn');
+    hooks.beforeEach(async function() {
+      await visit('/');
+      return await click('.spec-announce-btn');
     });
 
-    it("sets aria-live to 'assertive'", function() {
-      expect($('.spec-announcer').attr('aria-live')).to.equal('assertive');
+    test(`sets aria-live to 'assertive'`, function(assert) {
+      assert.dom('.spec-announcer').hasAttribute('aria-live', 'assertive');
     });
 
-    it("sets the content of the div", function() {
-      expect($('.spec-announcer').html().trim()).to.equal('This is a test');
+    test('sets the content of the div', function(assert) {
+      assert.dom('.spec-announcer').hasText('This is a test');
     });
+
   });
 
-  describe("changing the route change message", function() {
-    beforeEach(function() {
-      this.service.set('message', 'has finished loading');
-      return visit('/');
+  module('changing the route change message', function(hooks) {
+    hooks.beforeEach(async function() {
+      this.owner.lookup('service:announcer').set('message', 'has finished loading');
+      return await visit('/');
     });
 
-    it("updates the message", function() {
-      expect($('.spec-announcer').html().trim()).to.equal('Index title has finished loading');
+    test('updates the message', function(assert) {
+      assert.dom('.spec-announcer').hasText( 'Index title has finished loading');
     });
 
   });
 
-  describe("visiting the 'another' route", function() {
-    beforeEach(function() {
-      return visit('/another');
+  module(`visiting the 'another' route`, function(hooks) {
+    hooks.beforeEach(async function() {
+      return await visit('/another');
     });
 
-    it('pushes changes to the div', function() {
-      expect($('.spec-announcer').html().trim()).to.equal('Another title has loaded');
+    test('pushes changes to the div', function(assert) {
+      assert.dom('.spec-announcer').hasText('Another title has loaded');
     });
   });
 
-  describe("visiting the 'index' route", function() {
-    beforeEach(function() {
-      return visit('/');
+  module(`visiting the 'index' route`, function(hooks) {
+    hooks.beforeEach(async function() {
+      return await visit('/');
     });
 
-    it('pushes changes to the div', function() {
-      expect($('.spec-announcer').html().trim()).to.equal('Index title has loaded');
+    test('pushes changes to the div', function(assert) {
+      assert.dom('.spec-announcer').hasText('Index title has loaded');
```

Comment:
Awh, not a fan of rewriting these to qunit, but I don't really maintain this addon anymore 🤷‍♂️